### PR TITLE
Druid query runner

### PR DESCRIPTION
**Pull request by rauchy**
Here's my attempt at support for running SQL queries against a Druid.

Since I pretty much followed [this guide](https://discuss.redash.io/t/creating-a-new-query-runner-data-source-in-redash/347) for creating this query runner, I'm left with a feeling that there's nasty code duplication (in `run_query` and `get_schema`) - should we find a way to reuse these?

### CODE REVIEW

File: redash/query_runner/druid.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    from pydruid.db import connect
+    enabled = True
+except ImportError:
+    enabled = False
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.utils import json_dumps, json_loads
+
+TYPES_MAP = {
+}
```
Comment (arikfr) [inline redash/query_runner/druid.py]: image

?

File: redash/query_runner/druid.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    "type": "number",
+                    "default": 8082
+                },
+                "scheme": {
+                    "type": "string",
+                    "default": "http"
+                }
+            },
+            "order": ['scheme', 'host', 'port'],
+            "required": ['host']
```
Comment (arikfr) [inline redash/query_runner/druid.py]: No username/password support?

File: redash/query_runner/druid.py (hunk 1 above)
Comment (kravets-levko) [reply redash/query_runner/druid.py]: As far as I understand, Druid supports only one custom data type - `OTHER/COMPLEX`; however, I'm not sure how to represent it in Redash

Comment (arikfr) [reply redash/query_runner/druid.py]: The types map is not only for custom stuff.

File: redash/query_runner/druid.py (hunk 2 above)
Comment (rauchy) [reply redash/query_runner/druid.py]: While there seems to be [support for basic auth](https://github.com/druid-io/pydruid/blob/master/pydruid/client.py#L40) in the client, the docs for Druid's "experimental" SQL dialect don't show anything regarding authentication. Perhaps they expect people to protect endpoints by other means?

Comment (arikfr) [reply redash/query_runner/druid.py]: 👍 OK, let's just keep as is and if someone asks for it, revisit.

**Comment by arikfr** [comment]
:shipit:

**Pull request merged**